src/pages/home/report/ReportActionItem.js

```diff
@@ -105,6 +106,29 @@ class ReportActionItem extends Component {
         this.toggleReaction = this.toggleReaction.bind(this);
     }
 
+    componentDidMount() {
+        const htmlText = this.props.action.message[0].html;
```

Comment:
```suggestion
        const htmlText = lodashGet(this.props.action, ['message', 0, 'html']);
```

I'd maybe use something like `lodashGet()` here in case there is a bad message.

Reply:
**suggestion:** we can create a utility function for this like:

`ReportActionUtils.containsLink(reportAction)`

Then here just have

```js
if (!ReportActionUtils.containsLink(this.props.action)) {
    return;
}

Report.expandURLPreview(this.props.report.reportID, this.props.action.reportActionID);
```